source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -609,8 +609,9 @@ CacheLookupStatus CacheFilter::cacheLookupStatus() const {
     case CacheEntryStatus::LookupError:
       return CacheLookupStatus::LookupError;
     }
-    PANIC("Unhandled CacheLookupStatus encountered when retrieving request cache status: " +
-          std::to_string(static_cast<int>(filter_state_)));
+    IS_ENVOY_BUG(absl::StrCat("Unhandled CacheEntryStatus encountered when retrieving request cache status: " +
+                              std::to_string(static_cast<int>(filter_state_))));
```

Comment:
Oh yeah, also on all the IS_ENVOY_BUGs, I *think* you don't have to StrCat inside them, the ENVOY_BUG macros take multiple parameters and I think operate on fmt::format principles, e.g. I think this one could be
```
IS_ENVOY_BUG("Unhandled CacheEntryStatus encountered when retrieving request cache status: {}", static_cast<int>(filter_state_));
```

Reply:
This might be true of ENVOY_BUG, but I can't get this to work with IS_ENVOY_BUG. It complains about too many parameters.